svc/pkg/servers/ops/server-create/src/lib.rs

```diff
@@ -0,0 +1,157 @@
+use futures_util::FutureExt;
+use proto::backend::{self, pkg::*};
+use rivet_operation::prelude::*;
+
+#[operation(name = "servers-server-create")]
+pub async fn handle(
+	ctx: OperationContext<servers::server_create::Request>,
+) -> GlobalResult<servers::server_create::Response> {
+	let resources = unwrap_ref!(ctx.resources).clone();
+	let runtime = unwrap!(ctx.runtime.clone());
+
+	let server_id = Uuid::new_v4();
+	let game_id = unwrap_ref!(ctx.game_id).as_uuid();
+	let cluster_id = unwrap_ref!(ctx.cluster_id).as_uuid();
+	let datacenter_id = unwrap_ref!(ctx.datacenter_id).as_uuid();
+
+	let create_ts = ctx.ts();
+
+	rivet_pools::utils::crdb::tx(&ctx.crdb().await?, |tx| {
```

Comment:
because this is performance sensitive, let's see if we can get this working as a CTE using `unnest` (for bulk insertions of columns) in order to minimize round trips to the db. ask me on a call tomorrow and try gpt to see if it can build it for you.